socs/agents/acu/agent.py

```diff
@@ -1173,8 +1274,8 @@ def line_batcher(lines, n=10):
 
     @ocs_agent.param('az_endpoint1', type=float)
     @ocs_agent.param('az_endpoint2', type=float)
-    @ocs_agent.param('az_speed', type=float)
-    @ocs_agent.param('az_accel', type=float)
+    @ocs_agent.param('az_speed', type=float, default=None)
+    @ocs_agent.param('az_accel', type=float, default=None)
```

Comment:
Update the method signature override to have `az_speed=None` and `az_accel=None`.